src/foam/nanos/http/AuthWebAgent.java

```diff
@@ -121,7 +123,7 @@ public Session authenticate(X x) {
 
     //
     // Support for Basic HTTP Authentication
-    // Redimentary testing: curl --user username:password http://localhost:8080/service/dig
+    // Rudimentary testing: curl --user username:password http://localhost:8080/service/dig?entityId=1234
```

Comment:
I don't know if there is a requirement I'm missing here, but entityId feels like a poor name for this parameter, something like "actAs=1234" or (su= or doas= to steal names from unix/openbsd)

Reply:
Okay, will make the change, thanks!

Reply:
I agree that `entity` was a poor name for the User being acted as because it's too general. But that should at least clarify why @deejes named it `entityId`.

Reply:
doas seems too similar to daos for me